<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>610</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>confirm</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>541</width>
      <height>261</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>A</string>
     </attribute>
     <widget class="QLineEdit" name="form_rcptto">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>146</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>MAIL FROM</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="form_mailfrom">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>146</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>60</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>RCPT TO</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>B</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>C</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>30</y>
      <width>78</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mail Server</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>63</y>
      <width>38</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>EHLO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="form_ehlo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>63</y>
      <width>451</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="form_mailserver">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>451</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="form_subject">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>377</y>
      <width>451</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>377</y>
      <width>52</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Subject</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>410</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Contents</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="form_data">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>451</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>603</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMail_Attack_v0_1">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionInformation"/>
    <addaction name="actionLicense"/>
   </widget>
   <addaction name="menuMail_Attack_v0_1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionInformation">
   <property name="text">
    <string>information</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>license</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>form_mailserver</tabstop>
  <tabstop>form_ehlo</tabstop>
  <tabstop>form_mailfrom</tabstop>
  <tabstop>form_rcptto</tabstop>
  <tabstop>form_subject</tabstop>
  <tabstop>form_data</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
